pkg/informer/event-handler_test.go

```diff
@@ -0,0 +1,154 @@
+/*
```

Comment:
The file name should be event_handler_test.go?

Reply:
Indeed, I kept some habits from c++